config-templates/application-integration-test.properties.ctmpl

```diff
@@ -0,0 +1,11 @@
+{{- with $host := (( "http://34.71.179.202" )) -}}
+{{- with $wsmBaseUrl := (( printf "%s/dev-terra-workspace-manager" $host )) -}}
+{{- with $wsmCreateWorkspaceUrl := (( printf "%s/api/workspaces/v1" $wsmBaseUrl )) -}}
+{{- with $serviceAccountEmail := (( "william.thunderlord@test.firecloud.org" )) -}}
+{{- with $serviceAccountFilePath := env "SERVICE_ACCOUNT_FILE_PATH" -}}
+
+it.wsmCreateWorkspaceUrl={{$wsmCreateWorkspaceUrl}}
+it.serviceAccountEmail={{$serviceAccountEmail}}
+it.serviceAccountFilePath={{$serviceAccountFilePath}}
+
+{{end}}{{end}}{{end}}{{end}}{{end}}
```

Comment:
until we support other envs, this template is a bit silly, but I like it as an example of how to do templating.

Reply:
Yes, that was the idea. Setup example going forward.